services/httpservice/client.go

```diff
@@ -30,6 +30,38 @@ func IsReservedIP(ip net.IP) bool {
 	return false
 }
 
+func IsOwnIP(ip net.IP) (bool, error) {
+	interfaces, err := net.Interfaces()
+
+	if err != nil {
+		return false, err
+	}
+
+	for _, interf := range interfaces {
```

Comment:
nit: why `interf` and not `interface`? (Or `ifi` as used by the `net` package.)

Reply:
`interface` is a reserved word in go, so I couldn't use it. `ifi` sounds alright though, but I thought `interf` might be more clear.

Reply:
Goodness, of course `interface` is reserved. </epicfail> ;P 🤕